client/src/components/about.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+class About extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      abridged: true,
+      summary: "",
+      shortSummary: ""
+    };
+    this.shortenAbout = this.shortenAbout.bind(this);
+  }
+
+  shortenAbout(string) {
+    let shortenedSummary = "";
+    let sentenceArr = string.split(".");
+    for (let i = 0; i < sentenceArr.length; i++) {
```

Comment:
Since you're always going to want the first 5 sentences, with this for loop you're still going through the entire array. 

1. Bail out when you reach the 5th
or preferred
2. Use splice (maybe split) and pull out what you need

Reply:
rewritten to bail on the 5th